packages/eddsa-poseidon/tests/index.test.ts

```diff
@@ -197,11 +198,12 @@ describe("EdDSAPoseidon", () => {
 
     it("Should derive a public key from N random private keys", async () => {
         for (let i = 0, len = 10; i < len; i += 1) {
-            const privateKey = crypto.randomBytes(32)
+            const privateKey = crypto.getRandomValues(32)
+            const bgPrivateKey = beBufferToBigInt(Buffer.from(privateKey))
 
-            const publicKey = derivePublicKey(privateKey)
+            const publicKey = derivePublicKey(bgPrivateKey)
 
-            const circomlibPublicKey = eddsa.prv2pub(privateKey)
+            const circomlibPublicKey = eddsa.prv2pub(beBigIntToBuffer(bgPrivateKey))
```

Comment:
```ts
const privateKey = crypto.getRandomValues(32)
const publicKey = derivePublicKey(privateKey)
const circomlibPublicKey = eddsa.prv2pub(privateKey)
```

Should this still work?

Reply:
Nope, you'll get an error on the `derivePublicKey(privateKey)` call.

```bash
Argument of type 'Uint8Array' is not assignable to parameter of type 'BigNumberish'.
  Type 'Uint8Array' is missing the following properties from type 'Buffer': write, toJSON, equals, compare, and 66 more.ts(2345)
```

Reply:
```ts
const privateKey = crypto.getRandomValues(32)
const publicKey = derivePublicKey(Buffer.from(privateKey))
const circomlibPublicKey = eddsa.prv2pub(privateKey)
```

? Just to avoid converting buffer to bigint and vice versa.

Reply:
```ts
            const privateKey = crypto.getRandomValues(32)
            const publicKey = derivePublicKey(Buffer.from(privateKey))
            const circomlibPublicKey = eddsa.prv2pub(Buffer.from(privateKey))
```

This is correct

Reply:
Great, then also:
```ts
            const privateKey = Buffer.from(crypto.getRandomValues(32))
            const publicKey = derivePublicKey(privateKey)
            const circomlibPublicKey = eddsa.prv2pub(privateKey)
```